report-service/build.gradle

```diff
@@ -38,7 +41,30 @@ jar {
 	}
 }
 
+task jaxb {
+	System.setProperty('javax.xml.accessExternalSchema', 'all')
+	def jaxbTargetDir = file("src/main/java")
+	doLast {
+		jaxbTargetDir.mkdirs()
+		ant.taskdef(
+				name: 'xjc',
+				classname: 'com.sun.tools.xjc.XJCTask',
+				classpath: configurations.jaxb.asPath
+		)
+		ant.jaxbTargetDir = jaxbTargetDir
+		ant.xjc(
+				destdir: '${jaxbTargetDir}',
+				package: 'gov.cdc.dataingestion.nbs.jaxb',
+				schema: 'src/main/resources/xsd/PHDC.xsd'
+		)
+	}
+}
+
+compileJava.dependsOn jaxb
+
 dependencies {
+	implementation files('../hl7-parser/build/libs/hl7-parser.jar')
```

Comment:
Duc already merged dependency resolution on hl7-parser into main branch. 

I  would prefer to single line like 

implementation 'gov.cdc.dataingestion:hl7-parser:0.0.1-SNAPSHOT'


if possible.